Round robin_JB (#147)
jaybee84/ml-in-rd: content/06.multiple-approaches-required.md

### Using composite approaches can be a powerful strategy

We have described multiple approaches for maximizing the success of ML applications in rare disease, but it is rarely sufficient to use any of these techniques in isolation. 
Below, we highlight two recent works in the rare disease domain that draw on concepts of feature-representation-transfer, use of prior data, and regularization.

A large public dataset of acute myeloid leukemia (AML) patient samples with no drug response data and a small _in vitro_ experiment with drug response data form the basis of our first example [@doi:10.1038/s41467-017-02465-5].

> Training an ML model on the small _in-vitro_ dataset alone faced the _curse of dimensionality_ and was prohibitive toward representation learning.

Reviewer: ```suggestion
Training an ML model on the small _in vitro_ dataset alone faced the _curse of dimensionality_ and the dataset size prohibited representation learning.
```

"In-vitro" may also need to be fixed in a figure.

Dincer et al. trained a variational autoencoder on the large AML patient dataset (VAE; see [definitions]) to learn meaningful representations in an approach termed DeepProfile [@https://www.biorxiv.org/content/10.1101/278739v2] (Figure[@fig:4]a).

> The representations or _encodings_ learned by the VAE were then _transferred_ to the small _in-vitro_ dataset reducing it's number of features from thousands to eight, and increasing the performance of the final LASSO linear regression model.

Reviewer: ```suggestion
The representations or _encodings_ learned by the VAE were then _transferred_ to the small _in-vitro_ dataset reducing it's number of features from thousands to eight, and improving the performance of the final LASSO linear regression model.
```

In addition to improvement in performance, the _encodings_ learned by the VAE captured more biological pathways than PCA, which may be attributable to the constraints on the encodings imposed during the training process [definitions].

> Similar results were observed for prediction of histopathology in another rare cancer dataset[@https://www.biorxiv.org/content/10.1101/278739v2].

Reviewer: ```suggestion
Similar results were observed for prediction of histopathology in another rare cancer dataset [@https://www.biorxiv.org/content/10.1101/278739v2].
```

While DeepProfile was centered on training on an individual disease and tissue combination, some rare diseases affect multiple tissues that a researcher may be interested in studying together for the purpose of biological discovery. 
Studying multiple tissues poses significant challenges and a cross-tissue analysis may require an analyst to compare representations from multiple models.

> Moreover, models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.

Reviewer: ```suggestion
Models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.
```
